cSpell.json

```diff
@@ -0,0 +1,127 @@
+{
+  "language": "en",
+  "version": "0.1",
+  "words": [
+    "Linq",
+    "intellisense",
+    "nunit",
+    "NUnit",
+    "paramref",
+    "typeparam",
+    "typeparamref",
+    "wiki",
+    "ruleset",
+    "DWORD",
+    "Serializable",
+    "Parallelizable",
+    "nameof",
+    "typeof",
+    "nuget",
+    "VSIX",
+    "vstest",
+    "parallelization",
+    "hotfix",
+    "teamcity",
+    "runtimes",
+    "PDBs",
+    "Randomizer",
+    "unmanaged",
+    "prefilter",
+    "Guid",
+    "Guids",
+    "Enumerables",
+    "buildable",
+    "hotfixes",
+    "VSIXes",
+    "CODEPAGE",
+    "ints",
+    "ULPs",
+    "Ulps",
+    "nuspecs",
+    "combinatorially",
+    "Remoting",
+    "Hashtable",
+    "writability",
+    "Pluggable",
+    "Obsoletion",
+    "ReSharper",
+    "Dogfood",
+    "Silverlight",
+    "seamlessness",
+    "composability"
+  ],
+  "ignoreWords": [
+    "addin",
+    "addins",
+    "langword",
+    "ushort",
+    "ulong",
+    "iarray",
+    "sarray",
+    "idict",
+    "onetimesetup",
+    "onetimeteardown",
+    "testfixture",
+    "setupfixture",
+    "datapoint",
+    "datapoints",
+    "sbyte",
+    "IntelliTests",
+    "IntelliTest",
+    "shadowcopy",
+    "testcase",
+    "Testcase",
+    "netcoreapp",
+    "netcore",
+    "nunitlite",
+    "nunitagent",
+    "tracelevel",
+    "vssettings",
+    "notabug",
+    "norepro",
+    "wontfix",
+    "easyfix",
+    "nupkg",
+    "Terje",
+    "Prouse",
+    "runsettings",
+    "aolszowka",
+    "Letterle",
+    "Fixtureparams",
+    "tipsandtricks",
+    "Musser",
+    "Sandstrom",
+    "Idfor",
+    "editorconfig",
+    "Testgenerator",
+    "HKCU",
+    "testcasecount",
+    "inputfiles",
+    "inprocess",
+    "testlist",
+    "configfile",
+    "appbase",
+    "classname",
+    "runstate",
+    "fullname",
+    "teardowns",
+    "Halvorsen",
+    "iunequal",
+    "cppsample",
+    "SSCLI",
+    "Afterxxx",
+    "activeconfig",
+    "netstandard",
+    "Gallio",
+    "Codice",
+    "mytest",
+    "Rebracer",
+    "myassembly"
+  ],
+  "ignoreRegExpList": [
+    "\\((.*)\\)", // Markdown links
```

Comment:
📓 I know that comments in JSON aren't correct but this tooling allows us to get away with it and I figured the extra context was worth it.